WEBPAGE/main.py

```
@@ -0,0 +1,911 @@
+    requested_url = request.url
+    method = request.method
+    print(colorama.Fore.RED + f"405 Method Not Allowed: {method} {requested_url}" + colorama.Style.RESET_ALL) # for testing
+    logger.error(f"405 Method Not Allowed: {method} request to {requested_url}")
+    return f"Method {method} not allowed for {requested_url}", 405
+
+
+#start the app
+if __name__ == '__main__':
+    #logger.info(f"Starting app on http://{my_ip}:5000")
+    #app.run(host=my_ip, port=5000, debug=True, threaded=True)
+    logger.info(f"Starting app on http://localhost:8000")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)